hugr-passes/src/lib.rs

```diff
@@ -17,10 +17,34 @@ pub use inline_funcs::inline_acyclic;
 pub mod linearize_array;
 pub use linearize_array::LinearizeArrayPass;
 pub mod lower;
-pub mod merge_bbs;
 mod monomorphize;
+pub mod normalize_cfgs;
 pub mod untuple;
 
+/// Merge basic blocks. Subset of [normalize_cfgs], use the latter.
+#[deprecated(note = "Use normalize_cfgs", since = "0.15.1")]
+pub mod merge_bbs {
+    use hugr_core::hugr::{hugrmut::HugrMut, views::RootCheckable};
+    use hugr_core::ops::handle::CfgID;
+
+    /// Merge any basic blocks that are direct children of the specified CFG
+    /// i.e. where a basic block B has a single successor B' whose only predecessor
+    /// is B, B and B' can be combined.
+    ///
+    /// # Panics
+    ///
+    /// If the `entrypoint` of `cfg` is not an [OpType::CFG]
+    ///
+    /// [OpType::CFG]: hugr_core::ops::OpType::CFG
+    #[deprecated(note = "Use version in normalize_cfgs", since = "0.15.1")]
```

Comment:
Mixed up the version with tket's
```suggestion
    #[deprecated(note = "Use version in normalize_cfgs", since = "0.23.0")]
```